compose/material3/material3/build.gradle

```diff
@@ -220,6 +220,7 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
 
             configureEach {
                 languageSettings.optIn("androidx.compose.material3.ExperimentalMaterial3Api")
+                languageSettings.optIn("androidx.compose.ui.ExperimentalComposeUiApi")
```

Comment:
It shouldn't be here. In general usage of this APIs are prohibited, we do have some exceptions, but it shouldn't be by default